sorc/global_cycle.fd/read_write_data.f90

```diff
@@ -161,18 +169,21 @@ subroutine write_data(lensfc,idim,jdim,lsoil, &
  character(len=3)  :: rankch
 
  integer           :: myrank, error, ncid, id_var
+ integer           :: varid_slc, varid_stc
+ integer           :: dim_soil
 
  call mpi_comm_rank(mpi_comm_world, myrank, error)
 
  write(rankch, '(i3.3)') (myrank+1)
 
- fnbgso = "./fnbgso." // rankch
```

Comment:
This is an example of how the file names are specified with the rank on the end. In some cases the rank is the file, in others it's the ensemble member.

Reply:
Ignore per discussion.